src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ChangeWrapperHandleStrengthTests.Windows.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices.Tests.Common;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public partial class ChangeWrapperHandleStrengthTests
+    {
+        public static IEnumerable<object[]> ChangeWrapperHandleStrength_ComObject_TestData()
+        {
+            yield return new object[] { new ComImportObject() };
+
+            yield return new object[] { new DualComObject() };
+            yield return new object[] { new IUnknownComObject() };
+            yield return new object[] { new IDispatchComObject() };
+            yield return new object[] { new IInspectableComObject() };
+
+            yield return new object[] { new NonDualComObject() };
+            yield return new object[] { new AutoDispatchComObject() };
+            yield return new object[] { new AutoDualComObject() };
+
+            yield return new object[] { new NonDualComObjectEmpty() };
+            yield return new object[] { new AutoDispatchComObjectEmpty() };
+            yield return new object[] { new AutoDualComObjectEmpty() };
+        }
+
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [MemberData(nameof(ChangeWrapperHandleStrength_ComObject_TestData))]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Do we still need this ?

Reply:
I would assume so for the `IDispatch` at least.

Reply:
The testcase name itself contains windows.cs, then [PlatformSpecific(TestPlatforms.Windows)] isn't needed.

Reply:
Oh... sorry. That makes perfect sense.